ui/imports/shared/status/StatusChatInput.qml

```diff
@@ -1139,7 +1142,15 @@ Rectangle {
                 icon.name: "emojis"
                 type: StatusQ.StatusFlatRoundButton.Type.Tertiary
                 color: "transparent"
-                onClicked: togglePopup(emojiPopup, emojiBtn)
+                onClicked: {
+                    control.emojiPopupOpened = true
+                    // TODO fix clicking on the button when the popup is opened reopens it because the click outside closes it
+                    // emojiPopup.parent = control
+                    // emojiPopup.closePolicy = Popup.CloseOnEscape | Popup.CloseOnPressOutsideParent
```

Comment:
you commented this out right? should we create a bug for this if not already fixed?

Reply:
My bad, I'll try to fix right now